packages/pluggableWidgets/tree-view-web/src/components/ChevronIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { createElement, ReactElement } from "react";
+
+export const ChevronIcon = ({ className }: { className: string }): ReactElement => (
```

Comment:
Maybe we should move the icon to piw-internals as this is also being used in Accordion and Dropdown filter (If I am not wrong)